README.md

```diff
@@ -31,15 +49,40 @@ jobs:
         id: stark
         uses: stark-contrast/accessibility-check-action@0.2.0-beta.0
         with:
-            # Most of the following values are simply shell commands. You can use these to set up the container as needed for your app
-            token: ${{ github.event.inputs.token }} # The action will use this to send an audit report back to Stark.
-            setup: 'nvm install 16 && nvm use 16' # [Optional] Set up the container. Install some tools, export variables, etc.  
-            prebuild:  # [Optional] Run any prebuild steps, cd into subdirectories, etc.
-            build: 'npm run build' # [Optional] Build steps. Use && for multiple steps. 
-            serve:  'SERVER_PORT=3000 && npm run serve' # [Optional] Tell us how to serve your app. 
-            wait_time: 5000 # [Required, default 5000] Milliseconds to wait before your app can start serving
-            url: 'https://localhost' # [Required] Where does your app run? e.g. http://localhost:3000.
-            cleanup: # [Optional] After scanning, the command running in serve step is auto terminated. Use this to run any cleanup commands.
+            # [Required when used with Stark Projects]
```

Comment:
I flipping love this change in how this is documented. Much cleaner! The token here is the only thing that i think could give people pause. Perhaps my faith in people is lacking, but maybe we just need to say "remove this line if not using with Stark Projects"? Thoughts?